<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="vlMain">
   <item>
    <widget class="QWidget" name="shadowWidget" native="true">
     <property name="styleSheet">
      <string notr="true">#shadowWidget{
	background-color: rgb(255, 255, 255);
	border-radius:5px;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>layout learnning</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnMin">
          <property name="minimumSize">
           <size>
            <width>42</width>
            <height>42</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>42</width>
            <height>42</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border:none;
	background-color: rgb(255, 144, 116);
}

QPushButton:hover{
	background-color: rgb(255, 85, 0);
}

QPushButton:Press{
	background-color: rgb(255, 0, 0);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnMax">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>42</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>42</width>
            <height>42</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border:none;
	background-color: rgb(170, 255, 127);
}

QPushButton:hover{
	background-color: rgb(85, 255, 127);
}

QPushButton:Press{
	background-color: rgb(0, 255, 0);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnClose">
          <property name="minimumSize">
           <size>
            <width>42</width>
            <height>42</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>42</width>
            <height>42</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border:none;
	background-color: rgb(170, 255, 255);
	border-top-right-radius:5px;
}

QPushButton:hover{
	
	background-color: rgb(135, 255, 255);
}

QPushButton:pressed{
	
	background-color: rgb(85, 255, 255);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>10</number>
             </property>
             <item>
              <widget class="QLineEdit" name="lineEdit"/>
             </item>
             <item>
              <widget class="QPushButton" name="btnFile">
               <property name="text">
                <string>getFiles</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QProgressBar" name="progressBar">
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <family>新宋体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Md5</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <family>新宋体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>相同文件</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QListWidget" name="listWidgetMd5">
             <item>
              <property name="text">
               <string>新建项目</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="listWidgetDumplates"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
